packages/arb-token-bridge-ui/src/util/NetworkUtils.ts

```diff
@@ -0,0 +1,114 @@
+import { hexValue } from 'ethers/lib/utils'
+import { ChainId, getNetworkName, rpcURLs, getExplorerUrl } from './networks'
+import { BigNumber } from 'ethers'
+import { Web3Provider } from '@ethersproject/providers'
+import EthereumLogo from '../assets/EthereumLogo.png'
+import ArbitrumOneLogo from '../assets/ArbitrumOneLogo.svg'
+import ArbitrumNovaLogo from '../assets/ArbitrumNovaLogo.png'
+
+/* 
+  The list which will be available for network selection in navbar-dropdowns plus
+  Icons and colors attached with our networks available for selection
+*/
+export const supportedNetworks: {
+  [key in ChainId]?: { img: string; btnThemeClass: string }
+} = {
+  [ChainId.Mainnet]: {
+    img: EthereumLogo,
+    btnThemeClass: 'bg-blue-arbitrum'
+  },
+  [ChainId.ArbitrumOne]: {
+    img: ArbitrumOneLogo,
+    btnThemeClass: 'bg-blue-arbitrum'
+  },
+  [ChainId.ArbitrumNova]: {
+    img: ArbitrumNovaLogo,
+    btnThemeClass: 'bg-brick-dark'
+  }
+}
```

Comment:
I get why you extracted this, but I think we can just repeat ourselves, because the usage in `NetworkSelectionContainer` and `UnsupportedNetworkContent` is a bit different and we lose flexibility by having it here.